src/ripple/net/impl/RPCCall.cpp

```diff
@@ -495,6 +495,53 @@ class RPCParser
         return rpcError (rpcINVALID_PARAMS);
     }
 
+    bool isValidJson2(Json::Value const& jv)
+    {
+        if (jv.isObject())
+        {
+            if (jv.isMember("jsonrpc") && jv["jsonrpc"] == "2.0" &&
+                jv.isMember("ripplerpc") && jv["ripplerpc"] == "2.0" &&
+                jv.isMember("id") && jv.isMember("method"))
```

Comment:
This set of strings should be defined as JSON static strings in JsonFields.h (if they're not already) and updated everywhere.